mappings/net/minecraft/entity/mob/MagmaCubeEntity.mapping

```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_1589 net/minecraft/entity/mob/MagmaCubeEntity
+	METHOD method_20678 canMagmaCubeSpawn (Lnet/minecraft/class_1299;Lnet/minecraft/class_1936;Lnet/minecraft/class_3730;Lnet/minecraft/class_2338;Ljava/util/Random;)Z
```

Comment:
Is there a reason this isnt `canSpawn`?

Reply:
Enigma doesn't allow to name it as `canSpawn` because there's a method with the same signature in a superclass (`SlimeEntity`) that I mapped as `canSpawn`.

Reply:
Ah makes sense, I assumed it was something like this.